data_linked_in_ssr_manual/start_ssr_4_times_linux.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# See the documentation, Sec. "Use Cases", for an explanation of what this script is doing.
```

Comment:
Also here the URL would be nice, wouldn't it?

Reply:
Updated